deploy/kubernetes/base/cm-get-linode-id.yaml

```diff
@@ -11,9 +11,12 @@ data:
     id="$(kubectl get node/"${NODE_NAME}" -o jsonpath='{.spec.providerID}')"
     if [[ ! -z "${id}" ]]; then
       echo "${id}"
-      echo -n "${id:9}" > /linode-info/linode-id
+      if [[  "${id}" =~ "linode://" ]]; then
+        echo -n "${id:9}" > /linode-info/linode-id
+      else
+        echo "Provider ID: ${id} does not have linode:// prefix."
+        echo "Not populating /linode-info/linode-id file"
+      fi
       exit 0
     fi
     echo "Provider ID not found"
-    # Exit here so that we wait for the CCM to initialize the provider ID
-    exit 1
```

Comment:
Should we leave this and on the other yaml in this PR to be consistent with https://github.com/linode/linode-blockstorage-csi-driver/pull/92/files#diff-8d3664375a0e74ade8c2851454cfd0f8ec2eae83551d413f39dbfbbe2b9f8068R282 or remove the `exit 1` from the other yaml as well?

Reply:
sorry. - i removed the exit 1 by mistake, will fix it.

Reply:
fixed